src/app/images.tsx

```diff
@@ -102,6 +108,7 @@ export default function Images({
             <h2 className="font-bold pt-5" id="image_selection">
                 Image Selection
             </h2>
+            {image_error && <div>Please only enter valid image types: jpg, jpeg, cr2</div>}
```

Comment:
Here too, I think we'll want to add TIFF.

Additionally, this error message seems to show up when I select a directory with batch processing (although it does still let me add the directory and proceed with generating the HDR image).